controllers/clusterresources/redisuser_controller.go

```diff
@@ -174,9 +174,9 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			userID := fmt.Sprintf(instaclustr.RedisUserIDFmt, clusterID, user.Namespace)
 			err = r.API.DeleteRedisUser(userID)
 			if err != nil {
-				l.Error(err, "Cannot delete Redis user", "user", user.Name)
+				l.Error(err, "Cannot delete Redis user on Instaclustr", "user", user.Name)
```

Comment:
What does `Cannot delete Redis user on Instaclustr` mean? We delete users from a cluster, isn't it?

Reply:
Yes, fixed